core/extraction/towns_fund_round_three.py

```diff
@@ -957,7 +959,8 @@ def extract_footfall_outcomes(df_input: pd.DataFrame, project_lookup: dict, prog
     if invalid_projects := relevant_projects - set(project_lookup.keys()):
         raise ValidationError(
             [
-                vf.InvalidOutcomeProjectFailure(invalid_project=project, section="Footfall Indicator")
+                vf.InvalidOutcomeProjectFailure(invalid_project=project, section="Footfall Indicator", index=None)
+                # TODO: add index for footfalls
```

Comment:
is this TODO addressed in any PR?

Reply:
It's a ticket in the backlog we will probably move to the upcoming sprint, we separated it out because footfall outcomes is a bit trickier than the rest to retain the indexes for.